presto-main/src/test/java/com/facebook/presto/block/BlockAssertions.java

```diff
@@ -821,4 +848,11 @@ private static String generateRandomStringWithLength(int length)
         ThreadLocalRandom.current().nextBytes(array);
         return new String(array, UTF_8);
     }
+
+    private static void verifyNullRate(float nullRate)
```

Comment:
this method can be replaced with `checkArgument (nullRate < 0 || nullRate > 1, "nullRate must be between 0 and 1 inclusive");` in the helper class